server/server.js

```diff
@@ -122,6 +122,11 @@ function handleRequest(request, response) {
 
   if (parts[0] === "") parts.shift();
 
+  // If we're not being asked for a file, then we should check permissions.
+  if(parsedUrl.pathname.indexOf(".") === -1) {
+    checkUserPermission(parsedUrl);
```

Comment:
It looks like you can escape permission check just by having a dot in your board name.

Reply:
This is true, I hadn't considered that board names with a dot in would be a valid name. Perhaps I should look into including the token query with the resources too, more secure this way too. I'm unfamiliar with the templating engine but it looks straight forward enough so will look into it

Reply:
I've checked and the current implementation does not support dots in board names anyway. 

I've improved the code anyway so it will now do a permissions check on any file extension that is not a js,css,svg,ico,png,jpg or gif